src/herder/HerderImpl.cpp

```diff
@@ -52,6 +52,11 @@ using namespace std;
 namespace stellar
 {
 
+// Roughly ~10 minutes of consensus
+constexpr uint32 const CLOSE_TIME_DRIFT_LEDGER_WINDOW = 120;
```

Comment:
Nit: `CLOSE_TIME_DRIFT_LEDGER_WINDOW_SIZE` might be a better name here, at first I thought this was a time based constant and was trying to figure out how 120 mapped to 10 minutes.

Reply:
Done.